setup.py

```diff
@@ -2,21 +2,35 @@
 # coding=utf-8
 from __future__ import unicode_literals, absolute_import, print_function, division
 
-from sopel import __version__
 import sys
 
 try:
-    from setuptools import setup
+    from setuptools import setup, __version__ as setuptools_version
 except ImportError:
     print(
         'You do not have setuptools, and can not install Sopel. The easiest '
         'way to fix this is to install pip by following the instructions at '
-        'http://pip.readthedocs.org/en/latest/installing.html\n'
+        'https://pip.readthedocs.io/en/latest/installing/\n'
         'Alternately, you can run sopel without installing it by running '
         '"python sopel.py"',
         file=sys.stderr,
     )
     sys.exit(1)
+else:
+    version_info = setuptools_version.split('.')
+    major = int(version_info[0])
+    minor = int(version_info[1])
+
+    if major < 30 or (major == 30 and minor < 3):
+        print(
+            'Your version of setuptools is outdated: version 30.3 or above '
+            'is required to install Sopel. You can do that with '
+            '"pip install -U setuptools"\n'
+            'Alternately, you can run sopel without installing it by running '
```

Comment:
Always capitalization that bites you, isn't it? 😉

```suggestion
            'Alternately, you can run Sopel without installing it by running '
```